internal/cli/arguments/completion.go

```diff
@@ -112,8 +112,11 @@ func GetInstallableCores() []string {
 	})
 	var res []string
 	// transform the data structure for the completion
-	for _, i := range platforms.SearchOutput {
-		res = append(res, i.GetMetadata().GetId()+"\t"+i.GetReleases()[i.GetLatestVersion()].GetName())
+	for _, i := range platforms.GetSearchOutput() {
+		if i.GetLatestCompatibleVersion() == "" {
+			continue
+		}
+		res = append(res, i.GetMetadata().GetId()+"\t"+i.GetLatestCompatibleRelease().GetName())
```

Comment:
We can spare the call to `i.GetLatestCompatibleRelease()` if we reuse the result of `i.GetLatestCompatibleRelease()` 

```suggestion
	for _, i := range platforms.GetSearchOutput() {
		if latest := i.GetLatestCompatibleVersion(); latest != nil {
			res = append(res, i.GetMetadata().GetId()+"\t"+latest.GetName())
		}
```